wpa_project/info/templates/info/rules.html

```diff
@@ -0,0 +1,150 @@
+{% extends 'base.html' %}
+
+{% block title %} Range Rules {% endblock %}
+
+{% block main %}
+
+    <h1 class="text-center"> Range Rules </h1>
+    <p>
+        <strong>This is a public archery range - all persons must without exception operate under the following rules.
+        </strong> These rules are mandatory and are for the protection and safety of all archers. If you observe unsafe
+        behavior, report it immediately to the Range Master, a Park Ranger or LAPD (LAPD Park Police at 213-978-4600).
+    </p>
+    <h2>Short Range (Covered Range)</h2>
+    <div class="ps-4">
+        <ul>
+            <li>Enter and shoot at your own risk</li>
+            <li>Archers are expected to be courteous and cooperative on the range at all times</li>
+            <li>Broadheads & crossbows are strictly prohibited</li>
+            <li>Dogs are not allowed near the shooting lines and must be on a leash at all times</li>
+            <li>No running or playing on the range</li>
+            <li>Smoking is not allowed on the range</li>
+            <li>No alcohol is permitted on the range</li>
+            <li>No loud talking, noises, or musical devices allowed</li>
+            <li>All shooting must be done from the 18 meter shooting line unless a WPA authorized class is in session</li>
+            <li>Archers must straddle the shooting line (one foot on each side)</li>
+            <li>Arrows must always point downrange</li>
+            <li>Skydraws are not allowed</li>
+            <li>When done shooting, step off the shooting line so that other archers know you are done</li>
+            <li>Instruction by WPA Authorized instructors ... </li>
+        </ul>
+        <h4>Whistle Commands</h4>
+        <p>
+            <strong>2 Whistle Blasts:</strong> Straddle the shooting line<br>
+            <strong>1 Whistle Blast:</strong> Begin Shooting<br>
+            <strong>3 Whistle Blasts:</strong> Retrieve arrows<br>
+            <strong>5+ Whistle Blasts:</strong> EMERGENCY! Cease fire, unload, and do not move. Wait for further instructions.
+        </p>
+        <h4>Verbal Commands</h4>
+        <p>
+            <strong>CLEAR:</strong> when all archers are done shooting, anyone may call CLEAR. All archers should repeat
```

Comment:
Super duper minor and nitpicky, but can we capitalize "when" here for consistency?